.changeset/nasty-snakes-refuse.md

```diff
@@ -0,0 +1,8 @@
+---
+"@wso2is/admin.identity-providers.v1": minor
+"@wso2is/admin.applications.v1": minor
+"@wso2is/admin.connections.v1": minor
+"@wso2is/console": minor
+---
+
+Update Login flow authenticator modal to use connection templates list via API
```

Comment:
Since this change doesn't introduce a new feature, shall we change `minor` to `patch`?

Reply:
This PR improves the add authenticator modal in login flow use connection templates list via API. Since this is a considerable change to the feature, release will be kept as `minor`